README.adoc

```diff
@@ -10,77 +10,117 @@ image::https://github.com/rh-impact/pl-heudiconv/actions/workflows/ci.yml/badge.
 
 ## Abstract
 
-`pl-heudiconv` is a link:https://chrisproject.org/[_ChRIS^] _ds_ plugin that organizes brain imaging data into structured directory layouts.
+`pl-heudiconv` is a link:https://chrisproject.org/[_ChRIS_^] ds plugin that organizes brain imaging data into structured directory layouts.
+
+## Description
+The original `heudiconv` CLI tool is a flexible DICOM converter for organizing brain imaging data into structured directory layouts.
+This `pl-heudiconv` plugin is a wrapper that makes the `heudiconf` CLI tool available as a link:https://chrisproject.org/[_ChRIS_^] plugin.
+
+When running the `pl-heudiconv` plugin, you must specify two directories:
+
+* The _input directory_, where the brain imaging data is stored in DICOM files (with the `.dcm` file extension).
```

Comment:
```suggestion
* The _input directory_, where the brain imaging data is stored in DICOM files.
```

It seems like it will check all files and determine if they're DICOM files regardless of the file extension.

Reply:
Done, thanks :)